images/peer/README.md

```diff
@@ -0,0 +1,146 @@
+# Quick reference
+
+-	**Where to get Docker help**:  
+	[the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](http://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
+
+-	**Where to get Hyperledger Fabric help**:  
+	[Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/latest/), [Stackoverflow](https://stackoverflow.com/questions/tagged/hyperledger-fabric), or [Rocket.Chat](https://chat.hyperledger.org/channel/fabric)
+
+-	**Where to get support**:
+  The Fabric images are provided for development and test purposes. Various vendors provide
+  supported offerings for production use.
+
+-	**Architectures available**: amd64
+
+-	**Tags available**: Architecture specific (e.g. `amd64-2.0.0`), release specific (e.g. `2.0.0`), and latest patch of minor release (e.g. `2.0`).
+
+-	**Image Dockerfile**:  
+	[Dockerfile location](https://github.com/hyperledger/fabric/blob/master/images/peer/Dockerfile)
+
+-	**Source of this description**:  
+	[Fabric github repository](https://github.com/hyperledger/fabric/blob/master/images/peer/README.md)
+
+# What is Hyperledger Fabric?
+
+Hyperledger Fabric is an enterprise-grade permissioned distributed ledger framework for developing solutions and applications. Its modular and versatile design satisfies a broad range of industry use cases. It offers a unique approach to consensus that enables performance at scale while preserving privacy.
+
+# How to use this image
+
+## Start a fabric peer
+
+You can run a Fabric peer container as follows:
+
+```console
+$ docker run -d --publish 7051:7051 \
+-v /tmp/fabric/config/peer0.org1.example.com:/etc/hyperledger/fabric \
+-v /tmp/fabric/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp:/etc/hyperledger/fabric/msp \
+-v /tmp/fabric/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls:/etc/hyperledger/fabric/tls \
+-v /tmp/fabric/data/peer0.org1.example.com:/var/hyperledger/production \
+--name peer0.org1.example.com hyperledger/fabric-peer:2.0 peer node start
```

Comment:
`peer node start` is the command the container is launched with via the Dockerfile, no need to pass it into the run command

Reply:
understood, but i think it helps people understand what is going on, e.g. they may want to override `peer node start` with something like `peer node reset` to reset the peer's ledger, or they may want to use the binaries instead.